gateway.go

```diff
@@ -52,11 +61,19 @@ func (s *gatewayResource) gatewayHandler(w http.ResponseWriter, r *http.Request)
 	if s.verbose {
 		log.Printf("%s Handling %s\n", r.Method, r.URL.Path)
 	}
+
+	metrics := s.metricsFactory.Create(metricsEventGatewayRequest)
+	metrics = s.metricsFactory.Create(metricsEventGatewayRequest)
```

Comment:
it's looks like a rudimental (three equal lines)

Reply:
Oops, meant to remove those. Fixed!